recipes-support/boost/boost/riscv32__NR_futex.patch

```diff
@@ -1,3 +1,11 @@
+Define __NR_futex in terms of __NR_futex_time64 for rv32
```

Comment:
Could this go in OE instead? It isn't RISC-V specific.

Reply:
yes it can perhaps more importantly it needs to be submitted upstream